fvtr/oprofile/oprofile.exp

```diff
@@ -129,6 +129,8 @@ if { ${TARGET64} } {
 	lappend wordsize_l 64
 }
 
+cd ${FULLPATH}
```

Comment:
`${FULLPATH}` usually has the value returned from pwd with a '/' in appended.
Could you elaborate how is this command helping to fix the bug, please?

Reply:
Oh! I understand what's happening now and how this is helping.
Have we changed the pwd of the tests?  Interesting...

Reply:
@tuliom Not that I'm aware of.